content/general/limits.textile

```diff
@@ -0,0 +1,306 @@
+| "Number of subscribers":#subscribers (per channel) |\4=. Unlimited |
+| "Presence members":#presence-members (per channel) | 50 | 200 | 200 | Custom |
+| "Channel creation rate":#channel-create (per second) | 20 | >=50 | >=50 | Custom |
+| "Message publish rate":#message-publish-rate (per channel) | 15 | 50 | 50 | Custom |
+
+h3(#subscribers). Number of subscribers
+
+There is no limit on the number of clients that can be subscribed to a channel.
+
+h3(#presence-members). Presence members
+
+The number of clients that can use the "presence feature":/realtime/presence in a channel is limited as a precaution. If a large number of clients are subscribed to presence events on a channel then the amount of messages can scale very quickly.
+
+As an example, consider 200 clients subscribed to presence events on a channel and all of them join and leave the presence set within a few minutes. This would result in the following messages:
+
+* 200 presence messages published for the enter event.
+* 200 x 200 (40,000) messages subscribed to for the enter events.
+* 200 presence messages published for the leave event.
+* 200 x 200 (40,000) presence messages subscribed to for the leave event.
+
+This highlights the potential for 80,400 messages to be sent in a very short space of time on a single channel.
+
+Any clients that attempt to join the presence set over the presence member limit will be rejected and the error code @91003@ will be returned.
+
+h3(#channel-create). Channel creation rate
+
+The channel creation rate is the number of channels that can be created per second.
+
+The limit is calculated based on the hard limit for "peak channels":#peak-channels, with a minimum value of 50 per second for self-service and business accounts.
+
+If the channel creation rate is exceeded, an "instantaneous rate limit":#instantaneous-rates will be applied to channel creation.
+
+h3(#message-publish-rate). Message publish rate
+
+The message rate is the maximum rate at which messages can be published on a channel.
+
+If the message publish rate is exceeded for a channel, an "instantaneous rate limit":#instantaneous-rates on publishing will be applied to that channel.
+
+h2(#message). Message limits
+
+Message limits relate to the number, rate and bandwidth of "messages":/realtime/messages on your account.
+
+|_=. Limit |_=. Free |_=. Self-service |_=. Business |_=. Enterprise |
+| "Message size":#message-size (KiB) | 16 | 64 | 64 | 256 |
+| "History TTL":#history-ttl (hours) | 24 | 72 | 72 | Custom |
+
+h3(#message-size). Message size
+
+The message size is the maximum size of a single published "message":/glossary#messages.
+
+Note that the average message size is set to 2KiB. Although a message can be published that is 16KiB, this will count as 8 messages towards your "message count":#messages.
+
+Attempting to publish a message in excess of the message size limit will fail and thd the error code @40009@ will be returned.
```

Comment:
"in excess of" -> "larger than"?

Also, typo: "thd" -> "the"

Reply:
Thanks on both counts.